code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm

```diff
@@ -679,29 +679,75 @@ Difficulty: Very Hard
 	environment_smash = ENVIRONMENT_SMASH_NONE
 	AIStatus = AI_OFF
 	stop_automated_movement = TRUE
+
+/mob/living/simple_animal/hostile/lightgeist/healing
 	var/heal_power = 5
 
+/mob/living/simple_animal/hostile/lightgeist/plantgeist
+	name = "plantgeist"
+	desc = "This small floating creature is a completely unknown form of life... being near it fills you with a sense of tranquility."
+	icon_state = "plantgeist"
+	icon_living = "plantgeist"
+	friendly = "shines on"
+	faction = list("plants")
+	initial_language_holder = /datum/language_holder/venus //they only understand sylvan (plant language)
+	maxHealth = 10 //tough enough to resist a punch or something small, since they cost a fair bit of favor.
+	health = 10
+	light_range = 6
+	ventcrawler = VENTCRAWLER_NONE //they dont really need to be ventcrawling
+	var/heal_power = 3
+
 /mob/living/simple_animal/hostile/lightgeist/Initialize()
 	. = ..()
 	remove_verb(src, /mob/living/verb/pulled)
 	remove_verb(src, /mob/verb/me_verb)
 	var/datum/atom_hud/medsensor = GLOB.huds[DATA_HUD_MEDICAL_ADVANCED]
 	medsensor.add_hud_to(src)
 
-/mob/living/simple_animal/hostile/lightgeist/AttackingTarget()
+/mob/living/simple_animal/hostile/lightgeist/healing/AttackingTarget()
 	. = ..()
 	if(isliving(target) && target != src)
 		var/mob/living/L = target
 		if(L.stat != DEAD)
 			L.heal_overall_damage(heal_power, heal_power)
 			new /obj/effect/temp_visual/heal(get_turf(target), "#80F5FF")
 
+/mob/living/simple_animal/hostile/lightgeist/plantgeist/AttackingTarget() //plantgeists can only heal plantlike stuff
+	. = ..()
+	if(isliving(target) && target != src)
+		var/mob/living/L = target
+		if(L.stat != DEAD)
+			if(("vines" in L.faction) || ("plants" in L.faction))
+				L.heal_overall_damage(heal_power, heal_power)
+				new /obj/effect/temp_visual/heal(get_turf(target), "#5bf563")
+
+/obj/effect/mob_spawn/plantgeist
+	name = "dormant plantgeist"
+	desc = "A strange plant creature. It seems to be peacefully sleeping and it's mere presence soothes your nerves."
+	icon = 'icons/mob/animal.dmi'
+	icon_state = "dormantplantgeist"
+	density = FALSE
+	anchored = FALSE
+
+	mob_type = /mob/living/simple_animal/hostile/lightgeist/plantgeist
+	mob_name = "plantgeist"
+	death = FALSE
+	roundstart = FALSE
+	short_desc = "You are a plantgeist, a peaceful creature summoned by a plant god"
+	flavour_text = "<b>Try to prevent plant creatures from dying, and listen to your summoner otherwise. You can also click a plantlike creature to heal them.</b>"
```

Comment:
```suggestion
	flavour_text = "<b>Try to prevent plant creatures from dying and listen to your summoner otherwise. You can also click a plantlike creature to heal them.</b>"
```